### PR TITLE
Documentation for TVB REST server local setup

### PR DESCRIPTION
**Pull request by Spectre-ak**
Added guide for setting up local TVB rest server using keycloak.

### CODE REVIEW

File: framework_tvb/tvb/interfaces/rest/server/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```
+from tvb.interfaces.rest.client.tvb_client import TVBClient
+tvb_client = TVBClient("http://localhost:9090")
+tvb_client.browser_login()
+print(tvb_client.get_project_list())
+```
+
+
+
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/rest/server/README.md:90]: Include a phrase to inform the user that operations launched within the REST server, are visible in the GUI as well.

File: framework_tvb/tvb/interfaces/rest/server/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+If you've enabled keycloak login then after restart TVB will open the keycloak login window and you'll have to login with the username and password you've created earlier. 
+
+If TVB rest server started successfully then you'll be able to see the rest server documentation at [http://localhost:9090/doc/](http://localhost:9090/doc)
+
+Test the rest server, by firing a simulation from command line:
+
+```
+python -m tvb.interfaces.rest.client.examples.fire_simulation --rest-url=http://localhost:9090
+```
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/rest/server/README.md:78]: This code is also part of tvb-rest-client, so we can move it below line 80.

File: framework_tvb/tvb/interfaces/rest/server/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        ![This is an image](./keycloak_configs/keycloak_setting_rest_and_gui.jpg)
+
+    Click `Apply` to save changes and restart TVB.
+
+
+If you've enabled keycloak login then after restart TVB will open the keycloak login window and you'll have to login with the username and password you've created earlier. 
+
+If TVB rest server started successfully then you'll be able to see the rest server documentation at [http://localhost:9090/doc/](http://localhost:9090/doc)
+
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/rest/server/README.md:73]: Include a guideline on how to install tvb-rest-client here

File: framework_tvb/tvb/interfaces/rest/server/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Test the rest server, by firing a simulation from command line:
+
+```
+python -m tvb.interfaces.rest.client.examples.fire_simulation --rest-url=http://localhost:9090
+```
+ 
+Or using [tvb-rest-client](https://pypi.org/project/tvb-rest-client/)
+
+```
+from tvb.interfaces.rest.client.tvb_client import TVBClient
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/rest/server/README.md:83]: This example could be described as "using the tvb-rest-client API"

**Review by popaula937 — COMMENTED** [review]
Thank you for your contribution! 
It looks informative to me, I added a few suggestions for improvement.

Also, maybe the **keycloak_configs** folder would be better called **dev_resources** to differentiate it from the actual code.

File: framework_tvb/tvb/interfaces/rest/server/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+# TVB REST Server
+
+### TVB REST services which can be used to interact with TVB. 
+
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/rest/server/README.md:4]: Include a subtitle like: "Setup TVB REST server locally"

**Review by liadomide — COMMENTED** [review]
For now this looks ok, but we should plan to integrate it into tvb-documentation also

**Review by popaula937 — APPROVED** [review]

**Pull request merged**